DxDispatch/src/dxdispatch/DmlDispatchable.cpp

```diff
@@ -13,9 +15,23 @@ DmlDispatchable::DmlDispatchable(
     const Model::DmlDispatchableDesc& desc,
     const Dispatchable::Bindings& initBindings,
     IDxDispatchLogger* logger
-    ) : m_name(name), m_device(device), m_desc(desc), m_initBindings(std::move(initBindings)), m_logger(logger)
+    ) : m_name(name), m_device(device), m_desc(desc), 
+        m_initBindings(std::move(initBindings)), m_logger(logger), m_isSerializedGraph(false)
+{
+    if (std::holds_alternative<Model::DmlDispatchableDesc>(m_desc))
```

Comment:
We don't need this check because it is guaranteed that `desc` will always be of type DmlDispatchable desc.